pkg/sdk/testint/tags_integration_test.go

```diff
@@ -281,3 +283,915 @@ func TestInt_TagsShowByID(t *testing.T) {
 		require.Equal(t, id2, e2.ID())
 	})
 }
+
+func TestInt_TagsAssociations(t *testing.T) {
+	client := testClient(t)
+	ctx := testContext(t)
+
+	awsBucketUrl := testenvs.GetOrSkipTest(t, testenvs.AwsExternalBucketUrl)
+	awsRoleARN := testenvs.GetOrSkipTest(t, testenvs.AwsExternalRoleArn)
+
+	tag, tagCleanup := testClientHelper().Tag.CreateTag(t)
+	t.Cleanup(tagCleanup)
+
+	tagValue := "abc"
+	tags := []sdk.TagAssociation{
+		{
+			Name:  tag.ID(),
+			Value: tagValue,
+		},
+	}
+	unsetTags := []sdk.ObjectIdentifier{
+		tag.ID(),
+	}
+
+	t.Run("TestInt_TagAssociationForAccount", func(t *testing.T) {
+		err := client.Accounts.Alter(ctx, &sdk.AlterAccountOptions{
+			SetTag: tags,
+		})
+		require.NoError(t, err)
+
+		returnedTagValue, err := client.SystemFunctions.GetTag(ctx, tag.ID(), testClientHelper().Ids.AccountIdentifierWithLocator(), sdk.ObjectTypeAccount)
+		require.NoError(t, err)
+		assert.Equal(t, tagValue, returnedTagValue)
+
+		err = client.Accounts.Alter(ctx, &sdk.AlterAccountOptions{
+			UnsetTag: unsetTags,
+		})
+		require.NoError(t, err)
+
+		_, err = client.SystemFunctions.GetTag(ctx, tag.ID(), testClientHelper().Ids.AccountIdentifierWithLocator(), sdk.ObjectTypeAccount)
+		require.ErrorContains(t, err, "sql: Scan error on column index 0, name \"TAG\": converting NULL to string is unsupported")
+	})
+
+	accountObjectTestCases := []struct {
+		name        string
+		objectType  sdk.ObjectType
+		setupObject func() sdk.AccountObjectIdentifier
+		setTags     func(sdk.AccountObjectIdentifier, []sdk.TagAssociation) error
+		unsetTags   func(sdk.AccountObjectIdentifier, []sdk.ObjectIdentifier) error
+	}{
+		{
+			name:       "ApplicationPackage",
+			objectType: sdk.ObjectTypeApplicationPackage,
+			setupObject: func() sdk.AccountObjectIdentifier {
```

Comment:
nit: setupObject could be extracted to method:
- invoking creation on the given client method
- setting up the cleanup
- returning ID
setup of all tests would be shorter then

Reply:
Partially done.

Reply:
Any follow-up added to the cleanup epic?